sections/Lps/AmbassadorsBannerSliderSection.tsx

```diff
@@ -26,19 +26,27 @@ interface Props {
 
 const Styles = () => (
   <style>{`
-    .ambassadors-banner-slider-section .section-title {
+    .ambassadors-banner-slider-section .section-title,
+    .ambassadors-banner-slider-section .section-title h1 {
       font-weight: 400;
       padding: 0 12%;
       position: absolute;
       z-index: 2;
       color: #fff;
-      font-size: 48px;
+      font-size: 48px !important;
```

Comment:
@pauloquality Não é necessário `!important` aqui.

```suggestion
      font-size: 48px;
```